ui/app/AppLayouts/Onboarding2/pages/KeycardIntroPage.qml

```diff
@@ -192,22 +203,47 @@ KeycardBasePage {
             }
         },
         State {
-            name: "locked"
```

Comment:
I wondering about decomposing that component into smaller per-state chunks. Having multiple states with multiple property changes generates a lot of changes when adding sth there.

Reply:
I'm adding the 3 options to SB:

![image](https://github.com/user-attachments/assets/6962e630-cd10-4b86-8156-657c21bff029)

Reply:
We have separate states now for `Onboarding.KeycardState.BlockedPIN` and `Onboarding.KeycardState.BlockedPUK`; with the 3 "unblock" buttons being togglable individually, as required by the various flows